cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "https://ui.circleci.com",
+  "defaultCommandTimeout": 10000
```

Comment:
seems like a long timeout

Reply:
default is 4000 i can put it back to default